translation/templates.rst

```diff
@@ -39,6 +39,10 @@ works when you use a placeholder following the ``%var%`` pattern.
 
     If you need to use the percent character (``%``) in a string, escape it by
     doubling it: ``{% trans %}Percent: %percent%%%{% endtrans %}``
+    
+.. deprecated:: 4.2
+
+    The %var% notation of placeholders no longer works translating in Twig templates. use ``{var}`` instead.  ``%`` characters also no longer need to be escaped. ``{`` & ``}`` also do not have to be escaped.
```

Comment:
can you not write `%` and `&` chars no longer need to be escaped? no avoid repeating?

also a new line break is needed after `templates.` IMO

Reply:
right, that's better.  no need to be so wordy